src/main/java/codeu/model/store/basic/SettingStore.java

```diff
@@ -0,0 +1,63 @@
+package codeu.model.store.basic;
+import codeu.model.data.Setting;
+
+import java.util.ArrayList;
+import java.util.List;
+import codeu.model.store.persistence.PersistentStorageAgent;
+
+public class SettingStore {
+    /**
+     * Singleton instance of UserStore.
+     */
+    private static SettingStore instance;
+
+    /**
+     * Returns the singleton instance of UserStore that should be shared between all servlet classes.
+     * Do not call this function from a test; use getTestInstance() instead.
+     */
+    public static SettingStore getInstance() {
+        if (instance == null) {
+            instance = new SettingStore(PersistentStorageAgent.getInstance());
+        }
+        return instance;
+    }
+
+    /**
+     * The PersistentStorageAgent responsible for loading Setting from and saving Users to Datastore.
+     */
+
+    private PersistentStorageAgent persistentStorageAgent;
+
+    //** The in list memory of the setting. */
+    private List<Setting > setting;
+    /**
+     * This class is a singleton, so its constructor is private. Call getInstance() instead.
+     */
+    private SettingStore(PersistentStorageAgent persistentStorageAgent) {
+        this.persistentStorageAgent = persistentStorageAgent;
+        setting = new ArrayList<>();
+    }
+    //
+    public void updateColor(Setting setting) {
```

Comment:
How does this updateColor?

Reply:
added one more method that firs adds the settings and then this method will update the settings.